### DISCUSSION TITLE
Showing a context menu on hold

### DISCUSSION THREAD

**Discussion by naissa12** [lit/lit#4562] [Q&A]
I  have the a list of thumbnails that are rendered as follows:

```
  <div
         class="color"
         style="background:${rgbToHex(color.value.r, color.value.g, color.value.b )}"
  >
      <context-menu></context-menu>
  </div>
```

When I hold on the color div, I want to show the context menu. Only on hold not hover. How can I do that using lit? Please note its more than one thumbnail that's rendered. So I need to show the context menu for the thumbnail that the person holds on.

**Comment by augustjk** [comment]
Answered here: https://stackoverflow.com/a/78071167/19498660
